### PR TITLE
Update Casper logo, Fast Company logo

### PR DESCRIPTION
**Pull request by josephco**

### CODE REVIEW

File: lib/storefront/award-fastcompany.svg [hunk 1]
```diff
@@ -1,43 +1,47 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!-- Generator: Adobe Illustrator 21.0.2, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!-- Generator: Adobe Illustrator 19.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
 <svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
-	 viewBox="0 0 100 80" style="enable-background:new 0 0 100 80;" xml:space="preserve">
+	 width="100px" height="49px" viewBox="0 0 100 49" enable-background="new 0 0 100 49" xml:space="preserve">
```
Comment (florida) [inline lib/storefront/award-fastcompany.svg:4]: are we keeping this width / heights?

Comment (josephco) [reply lib/storefront/award-fastcompany.svg:4]: Yeah it gets stripped out in the gulp task

**Comment by ckeisato** [comment]
Since we use the logo a lot, do you think we should do a quick check on the rails site to make sure it looks ok?

**Comment by josephco** [comment]
Probably need to check to be safe, but it shouldn't affect anything based on my initial inspection

**Comment by robwierzbowski** [comment]
@josephco I *just* changed this. What's the new logo look like?

**Comment by robwierzbowski** [comment]
image

If you need to change, you may need to fix the Rails app as well. This module and the homepage module are very fit and finish-ey.

**Comment by josephco** [comment]
@robwierzbowski We updated the artboard to have less space above and below (flush to the icon) for easier resizing. When I check the Casper logo on Rails, I'll check this too.

**Comment by robwierzbowski** [comment]
OK, cool! You'll prob have to futz with some CSS. Send me any PRs and I'll CR!

**Pull request merged**